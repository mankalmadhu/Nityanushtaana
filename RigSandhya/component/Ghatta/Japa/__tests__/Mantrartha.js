import React from 'react';
import ಮ೦ತ್ರಾರ್ಥ from '../Mantrartha';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya', () => 'ವಿಷಯ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');

test("ಮ೦ತ್ರಾರ್ಥ", () => {
  expect(Renderer.create(
    <ಮ೦ತ್ರಾರ್ಥ />
  ).toJSON()).toMatchSnapshot();

});