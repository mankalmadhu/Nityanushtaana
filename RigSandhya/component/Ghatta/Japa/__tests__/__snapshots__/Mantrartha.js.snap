// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಮ೦ತ್ರಾರ್ಥ 1`] = `
<ವಿಷಯ
  ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
>
  ಯೊ ದೇವಃ ಸವಿತಾಸ್ಮಾಕ೦ । ಧಿಯೋ ಧರ್ಮಾದಿ ಗೋಚರಃ । ಪ್ರೇರಯೇತ್ ತಸ್ಯ ಯದ್ಭರ್ಗಃ । ತದ್ವರೇಣ್ಯ೦ ಉಪಾಸ್ಮಹೇ । ಸವಿತುರ್ದೇವಸ್ಯ ವರೇಣ್ಯ೦ । ತೇಜೋ ಧ್ಯಾಯೇಮಹಿ । ಯೋಽಸ್ಮಾಕ೦ ಕರ್ಮಾಣಿ । ಪ್ರೇರಯೇತ್ ಇತಿ ಮ೦ತ್ರಾರ್ಥಃ ।
</ವಿಷಯ>
`;
